jax/_src/numpy/lax_numpy.py

```diff
@@ -5973,19 +5973,44 @@ def meshgrid(*xi: ArrayLike, copy: bool = True, sparse: bool = False,
   return output
 
 
-@custom_jvp
-@util.implements(np.i0)
 @jit
 def i0(x: ArrayLike) -> Array:
+  """Calculate modified Bessel function of first kind, zeroth order.
+
+  JAX implementation of :func:`numpy.i0`.
+
+  Args:
+    x: scalar or array. Specifies the argument of Bessel function.
+
+  Returns:
+    An array containing the corresponding vlaues of the modified Bessel function
+    of ``x``.
+
+  See also:
```

Comment:
You could also mention `jax.scipy.special.i0e`

Reply:
Done